navigation/navigation-common/src/androidTest/res/values/ids.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  Copyright 2021 The Android Open Source Project
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+<resources>
+    <item type="id" name="nav_id_reference" />
+    <item type="id" name="nav_id_reference_2" />
```

Comment:
Is this resource needed?

Reply:
ah, apologies -- that's from when I thought there was a `Reference[]` type. Will remove

Reply:
done in 1c76b48